devel/fastmod/Portfile

```diff
@@ -0,0 +1,125 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               cargo   1.0
+PortGroup               github  1.0
+
+github.setup            facebookincubator fastmod 0.4.3 v
+github.tarball_from     archive
+
+categories              devel textproc
+platforms               darwin
+license                 Apache-2
+maintainers             {@cideM gmail.com:fbeeres}
+description             A fast partial replacement for the codemod tool
+long_description        fastmod is a tool to assist you with large-scale codebase refactors. \
+                        It supports most of codemod's options but is more focused on improving \
+                        the use case \"I want to use interactive mode to make sure my regex is \
+                        correct, and then I want to apply the regex everywhere\".
+homepage                https://github.com/facebookincubator/fastmod
```

Comment:
Among other things, `github.setup` automatically sets the `homepage`.  You do not need this line because of your `github.setup` statement above.